giraffe/src/utils/getTextMetrics.ts

```diff
@@ -18,9 +20,18 @@ export const getTextMetrics = (font: string, text: string): TextMetrics => {
   document.body.appendChild(div)
 
   span.innerText = text
-
+  spanWithDash.innerText = '-'
+
+  // Text with the same number of characters:
+  //   when it includes a dash (negative number), it tends to be skinnier than
+  //   a positive number because a dash is not as wide as a single digit (most of the time).
+  //   Add padding when text has a dash by making dashes twice as wide.
+  const widthPadding =
```

Comment:
is there a cleaner way of doing this? i don't think we can rely on a dash being the same thing as a negative indicator. some platforms might not render `-1` consistently with other platforms

Reply:
Yes there is a cleaner way. No extra `<span>` necessary. We do need to check for a dash though, unfortunately. I will send up a new commit in the near future.